package.json

```diff
@@ -18,9 +18,11 @@
     "@react-navigation/native-stack": "^6.5.2",
     "@types/react-native-vector-icons": "^6.4.10",
     "axios": "^0.26.1",
+    "cocoapods": "^0.0.0",
```

Comment:
Waarvoor is dit? Dit is namelijk een package manager voor IOS die losstaat van NPM.